bcgov/cas-registration
# Chore/django simple history

Adding the django-simple-history package to track changes to our user-controlled data tables.
This will:
- Add an erc_history schema to hold all the _history tables
- Add _history tables to all user-controlled data tables
- On any change to those tables, the snapshot of the data will be recorded and timestamped in the _history table

To test this manually:
- Using the app (or at least the endpoint), make a change to any table.
- use psql or pgadmin to check that your change was saved to the corresponding <table>_history table
- (ex: `select * from erc_history.operation_history;` would show the change history for the operation table)
